WordPress/Classes/ViewRelated/Reader/ReaderDetailWebviewViewController.swift

```diff
@@ -240,3 +253,18 @@ extension ReaderDetailWebviewViewController: UIViewControllerTransitioningDelega
         return FancyAlertPresentationController(presentedViewController: presented, presenting: presenting)
     }
 }
+
+// MARK: - Navigation Delegate
+
+extension ReaderDetailWebviewViewController: WKNavigationDelegate {
+    func webView(_ webView: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void) {
+        if navigationAction.navigationType == .linkActivated {
+            if let url = navigationAction.request.url {
+                coordinator?.handle(url)
+            }
+            decisionHandler(.cancel)
```

Comment:
Why .cancel ?

Reply:
`.cancel` tells the WebView to not open this link. Since we handle all the links, we don't let the WebView open then.